homepage/homepage/content/docs/groups/inheritance.mdx

```diff
@@ -1,153 +1,190 @@
 import { ContentByFramework, CodeGroup } from '@/components/forMdx'
 
-export const metadata = { 
-  description: "Create Groups that inherit members from other Groups."
+export const metadata = {
+  description: "Add groups as members of other groups."
 };
 
-# Group Inheritance
+# Groups as members
 
-Groups can inherit members from other groups using the `extend` method. 
+Groups can be added to other groups using the `addMember` method.
 
-When a group extends another group, members of the parent group will become automatically part of the child group.
+When a group is added as a member of another group, members of the parent group will become part of the child group.
 
-## Basic Usage
+## Basic usage
 
-Here's how to extend a group:
+Here's how to add a child group to a parent group:
```

Comment:
Child/parent doesn't sound aligned with the interpretation of groups as members.

How about wording it in something like:

> Here's how to add a group as member of another group